modules/tinymce/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - When loading content CSS takes more than half a second, the editor will be set to the "in progress" state until the CSS is ready. #TINY-10008
 
 ### Fixed
+- Clicking in the bullist toolbar button on a selected checklist would turn the list into paragraphs instead of a bulleted list. #TINY-9975
```

Comment:
Maybe something like:

```suggestion
- Applying an ordered or unordered list to a selected checklist would incorrectly turn the list into paragraphs. #TINY-9975
```

Since it's more broad than just the toolbar button it could in theory be applied with commands, menus etc.